apps/console/src/features/identity-providers/components/wizards/iproov/iproov-authentication-provider-create-wizard.tsx

```diff
@@ -0,0 +1,472 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentityAppsError } from "@wso2is/core/errors";
+import { AlertLevels, IdentifiableComponentInterface, TestableComponentInterface } from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import {
+    DocumentationLink,
+    GenericIcon,
+    Heading,
+    LinkButton,
+    PrimaryButton,
+    useDocumentation,
+    useWizardAlert
+} from "@wso2is/react-components";
+import { ContentLoader } from "@wso2is/react-components/src/components/loader/content-loader";
+import isEmpty from "lodash-es/isEmpty";
+import React, { FunctionComponent, ReactElement, Suspense, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch } from "react-redux";
+import { Dispatch } from "redux";
+import { Grid } from "semantic-ui-react";
+import {
+    IproovAuthenticationProviderCreateWizardContent
+} from "./iproov-authentication-provider-create-wizard-content";
+import { identityProviderConfig } from "../../../../../extensions/configs";
+import {
+    EventPublisher,
+    ModalWithSidePanel
+} from "../../../../../features/core";
+import { TierLimitReachErrorModal } from "../../../../core/components/tier-limit-reach-error-modal";
+import { createIdentityProvider } from "../../../api";
+import { getIdPIcons } from "../../../configs/ui";
+import { IdentityProviderManagementConstants } from "../../../constants";
+import {
+    GenericIdentityProviderCreateWizardPropsInterface,
+    IdentityProviderInterface
+} from "../../../models";
+
+/**
+ * Proptypes for the Iproov Authentication Provider Create Wizard.
+ */
+interface IproovAuthenticationProviderCreateWizardPropsInterface extends TestableComponentInterface,
```

Comment:
We don't need to use TestableComponentInterface for new components as it is now deprecated. Using IdentifiableComponentInterface and data-componentid would be enough.